gitpod-io/gitpod
# [supervisor] enable PWA

#### What it does

- It allows to start a new workspace as an app to enable desktop like experience.

#### How to test

- Switch to Code
- Start a workspace
- You should see start a PWA icon in the right corner of URl. See https://support.google.com/chrome/answer/9658361?co=GENIE.Platform%3DDesktop&hl=en
- Click it, try to open a file and user Ctrl+W to check whether it closes the editor, not the window.
